quipucords/scanner/task.py

```diff
@@ -34,17 +34,19 @@ def __init__(self, scan_job, scan_task):
             # partial results and start over.
             self.scan_task.connection_result.systems.all().delete()
 
-    def run(self):
+    def run(self, manager_interrupt):
         """Block that will be executed.
 
         Results are expected to be persisted.  The state of
         self.scan_task should be updated with status COMPLETE/FAILED
         before returning.
 
+        :param manager_interrupt: Shared memory Value which can inform
```

Comment:
I like that you added the param def here. Not a blocker but could consider adding it for the other places `manager_interrupt` is passed in.